Makefile

```diff
@@ -39,7 +39,7 @@ CGO_ENABLED ?= 1 # only used to compile server
 
 HAS_GO = $(shell hash go > /dev/null 2>&1 && echo "GO" || echo "NOGO" )
 ifeq ($(HAS_GO),GO)
-	XGO_VERSION ?= go-1.20.x
+	XGO_VERSION ?= 1.22.8
```

Comment:
Sure that this change still works?

Reply:
I don't think so, see https://pkg.go.dev/src.techknowlogick.com/xgo#readme-go-releases